views/pricing.html

```diff
@@ -144,12 +166,20 @@ <h1 class="text-center">Premium Data</h1>
           <td><i class="fas fa-check" aria-hidden="true"></i></td>
         </tr>
         <tr class="signup pricing">
-          <td></td>
-          <td><a href="/login" class="btn btn-warning">Sign in</a></td>
+          <td  class="left-heading"></td>
+          <td><a href="/login" class="btn btn-primary">Sign in</a></td>
           <td><a href="mailto:support@datahub.io?subject=Interested in DataHub Premium Data Offer">Contact us</a></td>
         </tr>
       </tbody>
     </table>
+    <div class="hire-us">
+        <div class="alert alert-success" role="alert">
+          We can help you design, build or enhance your own data-infrastructure or data-driven project - <a href="/hire-us">hire us</a>.
```

Comment:
Can we make this message centred and full width?

Reply:
@anuveyatsu 
* Made the alerts full width
* removed old dashboard content
* fixed minor bugs on dashboard

the icons are working fine on my end, i would like you to take a look